service/src/main/java/com/commercetools/pspadapter/payone/mapping/PayoneRequestFactory.java

```diff
@@ -33,4 +36,31 @@ public CaptureRequest createCaptureRequest(final PaymentWithCartLike paymentWith
         throw new UnsupportedOperationException("this request type is not supported by this payment method.");
     }
 
+    protected void mapFormPaymentWithCartLike(final AuthorizationRequest request,
+                                              final PaymentWithCartLike paymentWithCartLike,
+                                              final Logger logger) {
+        final Payment ctPayment = paymentWithCartLike.getPayment();
+        final CartLike ctCartLike = paymentWithCartLike.getCartLike();
+
+        MappingUtil.mapCustomFieldsFromPayment(request, ctPayment.getCustom());
+
+
+        try {
+            MappingUtil.mapCustomerToRequest(request, ctPayment.getCustomer());
+        } catch (final IllegalArgumentException ex) {
+            logger.debug("Could not fully map payment with ID {} {}", paymentWithCartLike.getPayment().getId(), ex.getMessage());
+        }
+
+        try {
+            MappingUtil.mapBillingAddressToRequest(request, ctCartLike.getBillingAddress());
+        } catch (final IllegalArgumentException ex) {
+            logger.error("Could not fully map payment with ID {} {}", paymentWithCartLike.getPayment().getId(), ex.getMessage());
```

Comment:
The reason for different Log-Levels is, that only billing Address (or more specific the contained values Country and Lastname) is required. The others are optinal and sould not flood the papertrail logs.
A single try/catch could prevent the execution of the other mapping methods (case: `customer == null ` and `billingAddress != null`)